optaweb-employee-rostering-frontend/.eslintrc.json

```diff
@@ -61,6 +62,7 @@
         " "
       ]
     ],
+    "import/extensions": "off", // Suddenly, the new norm for imports is to include the file extension
```

Comment:
Please don't turn off the rule, rather override its configuration so that it matches the current code style. The rule should remain active to ensure consistent import style.

This works for me:
```json
"import/extensions": ["error", "never", {"css":  "always"}],
```